app/models/song.rb

```diff
@@ -29,4 +29,9 @@ def to_s
     title
   end
 
+  def full_label
+    ("<strong>#{title}</strong> " + 
+     [composed_by, arranged_by, published_by].join(' | ')).html_safe
+  end
```

Comment:
I Know this should go into a decorator, but I did not manage to register it correctly somehow.